snow/networking/router/chain_router.go

```diff
@@ -706,3 +714,92 @@ func (cr *ChainRouter) clearRequest(
 	cr.metrics.outstandingRequests.Set(float64(cr.timedRequests.Len()))
 	return uniqueRequestID, &request
 }
+
+func (cr *ChainRouter) handleSimplexMessage(ctx context.Context, msg message.InboundMessage) {
+	m := msg.Message()
+	nodeID := msg.NodeID()
+	op := msg.Op()
+
+	if msg.Op() != message.SimplexOp {
+		cr.log.Debug("dropping message with invalid field",
```

Comment:
why are we dropping because it has an invalid field? aren't we dropping it because the op isn't compatible?